internal/controller/controller.go

```diff
@@ -288,6 +293,69 @@ func (c *httpController) exportRecordedData(writer http.ResponseWriter, request
 // importRecordedData imports data from a previously exported record session.
 // An error is returned if a record or replay session is currently running or the data is incomplete
 func (c *httpController) importRecordedData(writer http.ResponseWriter, request *http.Request) {
-	//TODO implement me using TDD
-	writer.WriteHeader(http.StatusNotImplemented)
+	importedRecordedData := &dtos.RecordedData{}
+	var reader io.ReadCloser
+	var err error
+	var overWriteProfilesDevices bool
+
+	compression := request.URL.Query().Get("compression")
+	overWriteProfilesDevices, err = strconv.ParseBool(request.URL.Query().Get("overwrite"))
```

Comment:
`overwrite` is optional and defaults to `true`. What happens here when to provided.
https://docs.edgexfoundry.org/3.0/design/adr/application/0025-Record-and-Replay/#post_2